docroot/sites/ighn.international.uiowa.edu/modules/ighn_core/src/Entity/IghnPerson.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Drupal\ighn_core\Entity;
+
+use Drupal\uiowa_core\Entity\NodeBundleBase;
+use Drupal\uiowa_core\Entity\RendersAsCardInterface;
+
+/**
+ * Provides an interface for IGHN person entries.
+ */
+class IghnPerson extends NodeBundleBase implements RendersAsCardInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildCard(array &$build) {
+    parent::buildCard($build);
+
+    // Add the person library.
+    $build['#attached']['library'][] = 'uids_base/person';
+
+    // Process additional card mappings.
+    $this->mapFieldsToCardBuild($build, [
+      '#media' => 'field_ighn_person_image',
+      '#subtitle' => 'field_ighn_person_credentials',
+      // '#url' => 'field_ighn_person_cv_link',
```

Comment:
```suggestion

```
can this be removed?

Reply:
Yes. Maybe add a "IGHN Person card uses `field_ighn_person_cv_link` for link." bullet point to the IS?